MAUI/ComboBox/UI-Customization.md

```diff
@@ -0,0 +1,176 @@
+                    Placeholder="Select a social media" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+comboBox.PlaceholderColor = Colors.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox placeholder color](Images/UICustomization/PlaceholderColor.png)
+
+## Clear Button Icon Color
+
+The clear button icon color can be changed by using the `ClearButtonIconColor` property. The default value of `ClearButtonIconColor` property is `Colors.Black`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox x:Name="combobox"
+                    WidthRequest="250"
+                    ItemsSource="{Binding SocialMedias}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name"
+                    ClearButtonIconColor="Red" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+combobox.ClearButtonIconColor = Colors.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox clear button color](Images/UICustomization/ClearButtonColor.png)
+
+## DropDown Icon Color
+
+The DropDown icon color can be changed by using the `DropDownIconColor` property. The default value of `DropDownIconColor` property is `Colors.Black`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox x:Name="combobox"
+                    WidthRequest="250"
+                    ItemsSource="{Binding SocialMedias}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name"
+                    DropDownIconColor="Red" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+combobox.DropDownIconColor = Colors.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox  drop-down icon color](Images/UICustomization/DropDownIconColor.png)
+
+
+## Border Color
+
+The ComboBox border color can be changed by using the `BorderColor` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox x:Name="combobox"
+                    WidthRequest="250"
+                    ItemsSource="{Binding SocialMedias}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name"
+                    DropDownIconColor="Red"
+                    BorderColor="Red" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+combobox.BorderColor = Colors.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox border color](Images/UICustomization/BorderColor.png)
+
+
+## Maximum DropDown Height
+
+The maximum height of the drop-down can be changed by using the `MaxDropDownHeight` property of `ComboBox` control. The default value of `MaxDropDownHeight` property is `400d`. 
+
+ N> If the `MaxDropDownHeight` is too small compared to the populated items, scroll viewer will be automatically shown to navigate the hidden items.
```

Comment:
N> If the `MaxDropDownHeight` is too small compared to the populated items, the scroll viewer will be automatically shown to navigate the hidden items.